docs/design.md

```diff
@@ -38,13 +58,164 @@ spec:
 ```
 
 ### Command
+
 Optionally you can override the default command of a container by providing
 `command` in the Substratus resource:
 ```yaml
 spec:
   command: ["train.sh"]
 ```
 
+### User
+
+All containers should use the `ml` user for all training, importing, notebook, etc operations. The `/home/ml` directory should correspond to the `WORKDIR` of all Dockerfiles.
```

Comment:
This would require updating all images and retesting, which I think is fine and rather do it earlier than later. Just making a note here to make sure this happens.

Reply:
A few things that would make this churn take less effort: 1. Using a base substratus image, 2. Using a monorepo for our models.

Reply:
Im starting work on 1: https://github.com/substratusai/substratus/issues/61

Reply:
Created an issue to track / discuss number 2: https://github.com/substratusai/substratus/issues/106

Reply:
Updated to remove root requirement.